### PR TITLE
Item feedback form

### PR DESCRIPTION
**Pull request by jwillbpl**
#100

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -53,4 +53,8 @@
   resources :downloads, only: [:show]
   get 'start_download/:id', to: 'downloads#trigger_download', as: 'trigger_downloads'
   get 'start_zip_download/:id', to: 'zip_downloads#trigger_download', as: 'trigger_zip_downloads'
+
+  # question form
+  match 'item_feedback', to: 'feedback#item', via: [:get, :post]
+  # get 'feedback/complete', to: 'feedback#complete'
```
Comment (ebenenglish) [inline config/routes.rb:59]: remove this unused line

File: app/controllers/feedback_controller.rb [hunk 2]
```diff
@@ -11,6 +11,32 @@ def show
...
+    @errors = []
+
+    if request.post? && validate
+      Notifier.feedback(params).deliver_now
+      flash[:success] = t('blacklight.feedback.complete.title')
+      respond_to do |format|
+        format.html do
+          return render 'item_success' if request.xhr?
+
+          redirect_to solr_document_path(params[:ark_id])
```
Comment (ebenenglish) [inline app/controllers/feedback_controller.rb:27]: Change this to:
```ruby
redirect_to solr_document_path(@document)
```

File: spec/features/catalog/item_feedback_form_spec.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+describe 'item feedback modal', js: true do
+  before(:each) do
+    visit solr_document_path(id: 'bpl-dev:df65v790j')
+  end
+
+  it 'displays the item feedback link when the page is loaded' do
+    expect(page).to have_selector('#itemFeedbackLink')
+  end
+
+  it 'opens up the modal when I click the button' do
```
Comment (ebenenglish) [inline spec/features/catalog/item_feedback_form_spec.rb:14]: Change this to:
```ruby
it 'renders the modal popup when the button is clicked' do
```

**Review by ebenenglish — CHANGES_REQUESTED** [review]
Looks good, just a couple small tweaks, thanks!

**Comment by jwillbpl** [comment]
Thank you, I pushed the changes!

**Review by ebenenglish — APPROVED** [review]

**Pull request merged**